### ISSUE TITLE
Cannot flash Ardupilot firmware to micro minimosd

### ISSUE THREAD

**Issue by Daddy-Boomalati** [ShikOfTheRa/scarab-osd#611]
I'm sorry to put the issue here, but I keep getting a ReCaptcha error on FPVLab.

I'm using the latest firmware and GUI builds and I keep getting an error. I can read and write to the board with whatever firmware came loaded on it, but I cannot flash it. Attached is the error message.

Thanks!
![Screenshot 2020-05-18 09 56 26](https://user-images.githubusercontent.com/7673403/82468729-27fc4100-9a91-11ea-8ccb-d8822c234f92.png)

**Comment by ShikOfTheRa** [comment]
FAQ # 1 Unable to flash and I get "Programming FAILED: sending 3020"

99% of time this is connectivity or power issue
Check you are selecting the correct comm port
TX of OSD must connect to RX of programmer
RX of OSD must connect to TX of programmer
Programmer requires all six pins connected
Nothing else must power the OSD when programming. Only the USB programmer

**Comment by ShikOfTheRa** [comment]
https://github.com/ShikOfTheRa/scarab-osd/wiki/FAQ

**Comment by Daddy-Boomalati** [comment]
Thank you!!!

Dave Smith

On May 21, 2020, at 03:59, ShikOfTheRa <notifications@github.com> wrote:

﻿

FAQ # 1 Unable to flash and I get "Programming FAILED: sending 3020"

99% of time this is connectivity or power issue
Check you are selecting the correct comm port
TX of OSD must connect to RX of programmer
RX of OSD must connect to TX of programmer
Programmer requires all six pins connected
Nothing else must power the OSD when programming. Only the USB programmer

**Comment by ShikOfTheRa** [comment]
Let me know if it works!

**Comment by Daddy-Boomalati** [comment]
Yeah, I don’t know what is up with it. I know I have it wired correctly. All six pins in the correct order. I can read and write to it, just not upload firmware. It connects fine, but the bootloader is DOA or something.

I even tried a different computer in case it was that. I am going to readymaderc tomorrow, and I’ll see if one of their guys can flash it for me, or exchange it out.

Thank you so much for taking the time to respond. You’ve always been a stand-up guy.

Dave Smith

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by ShikOfTheRa** [comment]
Any luck?

The only things I think could impact this:

1 board is powered up by something else. Or have large capacitors on power line.
2 the DTR pin is not connected end to end.
3 faulty board / programmer. Maybe it is set to 3.3v instead of 5v?
4 the minim does not have standard Arduino 56k bootloader. I only know of one manufacturer who did this and it wasn’t a minim style board.
5 the bootloader is corrupt – but I would expect it not to start up so extremely unlikely.

**Comment by Daddy-Boomalati** [comment]
I ordered a minim board off eBay and it worked fine. The only thing (in my small pool of knowledge) can think is that the micro board was corrupt. The minim board took right off, and actually fits my 3D-printed mount I made better anyway, so I’m going with it!

I really appreciate you taking the time to help me.

Thanks,
Dave

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
